app/models/user.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class User < ApplicationRecord
+  has_many :posts
```

Comment:
We should have a `dependent: :destroy` call here so that whenever a user is deleted the post will also be deleted. While we technically don't have any controllers here that would delete the project, for this particular association between user and posts we would like to remove all posts that a user created if that user was deleted from the database.